packages/notifier/test/test-notifier-adaptor.js

```diff
@@ -136,3 +140,211 @@ test('notifier adaptor - update from notifier fails', t => {
   const n = makeNotifierFromAsyncIterable(explodingStream);
   return observeNotifier(n, u);
 });
+
+// /////////////////////////////// makeNotifierFromSubscriber /////////////////////////////////
+
+/** @param {{conclusionMethod: 'finish' | 'fail', conclusionValue: any}} config */
+const makeGeometricPublishKit = ({ conclusionMethod, conclusionValue }) => {
```

Comment:
this sounds like it's making a PublicKit but it's returning something different.

```suggestion
const makeGeometricPublishHelper = ({ conclusionMethod, conclusionValue }) => {
```
Looks like calls have the same arguments. Why parameterize?
```suggestion
const makeGeometricPublishHelper = () => {
```
Also looks like all tests share this first line. Once this is in its own file, it would work well to put in a hook:
```
test.beforeEach(t => {
  Object.assign(t.context, makeGeometricPublishHelper());
}
```
Or even,
```
test.beforeEach(t => {
  Object.assign(t.context, makeGeometricPublishHelper());
  t.context.notifier = await makeNotifierFromSubscriber(t.context.subscriber);
}
```

Reply:
Renamed for clarity.

Reply:
Makes sense. I was wrong about the parameterization